docs/model_developers/handlers.rst

```diff
@@ -95,6 +95,17 @@ Each Handler should override 4 methods of the CRUDHandler:
 The context (See :class:`~inmanta.agent.handler.HandlerContext`) passed to most methods is used to
 report results, changes and logs to the handler and the server.
 
+The :meth:`~inmanta.agent.handler.HandlerContext.set_fact` method can be used to report facts to the server.
+By default, facts are considered stale after not being refreshed or updated for a certain time. This time span is
```

Comment:
I would first explain here there there are two models (fact push and fact pull), and add some example code on both. 

I would also explain: if you have a fact that never changes, do it like so. If it may change, do it like so. 

and explain it should not be used for things that change rapidly (e.g. cpu usage), as they are not intended to refresh very quickly.